ui/main/src/app/modules/share/archives-logging-filters/archives-logging-filters.component.scss

```diff
@@ -31,14 +31,6 @@
     border-color: var(--opfab-form-border-color);
 }
 
-.opfab-publish-date {
```

Comment:
If you remove .opfab-publish-date, you should also remove this : 
![image](https://github.com/user-attachments/assets/13cd57b4-9127-4cec-ad52-ca0fe76e9256)